ginmetrics/middleware.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 
-	"github.com/penglongli/gin-metrics/bloom"
+	"github.com/jiekun/gin-metrics/bloom"
```

Comment:
```suggestion
	"github.com/penglongli/gin-metrics/bloom"
```

I'm sorry that the modified code touched this line. Fixing now.